--- makepkg.conf	2024-03-16 12:40:33.261998792 +0200
+++ makepkg.conf	2024-03-16 12:40:33.261998792 +0200
@@ -50,7 +50,7 @@
 LTOFLAGS="-flto=auto"
 RUSTFLAGS="-Cforce-frame-pointers=yes"
 #-- Make Flags: change this for DistCC/SMP systems
-#MAKEFLAGS="-j2"
+MAKEFLAGS="-j$(($(nproc)+1))"
 #-- Debugging flags
 DEBUG_CFLAGS="-g"
 DEBUG_CXXFLAGS="$DEBUG_CFLAGS"
@@ -83,7 +83,7 @@
 #   These are default values for the options=() settings
 #########################################################################
 #
-# Makepkg defaults: OPTIONS=(!strip docs libtool staticlibs emptydirs !zipman !purge debug !lto !autodeps)
+# Makepkg defaults: OPTIONS=(!strip docs libtool staticlibs emptydirs !zipman !purge !debug !lto !autodeps)
 #  A negated option will do the opposite of the comments below.
 #
 #-- strip:      Strip symbols from binaries/libraries
